File: java-i/7.3.java
//7-3 逆序的三位数
/*
2021.10.09
19.25.40
By Xcl
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b=a/100;
        int c=(a%100)/10;
        int d=(a%100)%10;
        if(d==0){
            System.out.print(b+c*10);
        }
        else{
            System.out.print(b+c*10+d*100);
        }
    }
}

File: python-p/7.3.py
##7-3 逆序的三位数
'''
2021.10.09
19.25.40
By Xcl
'''
a = int(input())
b=int(a/100)
c=int((a%100)/10)
d=int((a%100)%10)
if(d==0):
    print(b+c*10)
else:
    print(b+c*10+d*100)
